### ISSUE TITLE
Cypress not installing binary in gitlab pipeline CI

### ISSUE THREAD

**Issue by isabellaslv** [cypress-io/cypress#30440]
### Current behavior

I'm getting crazy trying to install cypress in my gitlab pipeline. It works fine on my local (windows).
Even after running npm install cypress, the cache folder is not created
my gitlab yml file:

```
CYPRESS_CACHE_FOLDER: $PROJECT_DIR/.cache/Cypress

cache:
  paths:
    - ~/.cache/Cypress
    - .cache/Cypress
    - $PROJECT_DIR/.cache/Cypress


e2e-test:
  image: $CYPRESS_IMG
  script:
    - export HTTP_PROXY=[my proxy]
    - export HTTPS_PROXY=[my proxy]
    - npm config set registry [my registry]
    - npm set strict-ssl false
    - export CYPRESS_CACHE_FOLDER=$PROJECT_DIR/.cache/Cypress
    - export CYPRESS_CRASH_REPORTS=1
    - npm ci
    - npm install cypress
    - ./node_modules/.bin/cypress install --force
    - npx cypress run
```

The output:
![image](https://github.com/user-attachments/assets/83a15e03-aa82-49a5-ac08-7fbb89c50517)

### Cypress Version

13.15.0

### Node version

20

### Package Manager

npm

### Package Manager Version

idk but between 17 and 20

### Operating system

Linux

### Operating System Version

linux-x64 (Debian - 11)

### Other

I've already tried to cache everything, but even after running npm install cypress, the cache folder is not created. Other things I tried:

- set the artifacts;
- remove the proxy;
- remove registry;
- basically everything in this documentation: https://on.cypress.io/not-installed-ci-error

**Comment by MikeMcC399** [comment]
Duplicate of conversation in https://discord.com/channels/755913899261296641/1298725753973637190/1298725753973637190

The image in use is not a Cypress Docker image.

Feedback in Discord was:

> It looks like $PROJECT_DIR is not set in the workflow.
> To troubleshoot Cypress installation see https://docs.cypress.io/app/references/advanced-installation#Troubleshoot-installation
> export DEBUG=cypress:cli* run installation and check the logs. 
> It may be a problem with ownership and access rights.
